tmt/steps/discover/fmf.py

```diff
@@ -342,6 +342,37 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                 if not self.opt('dry'):
                     shutil.copytree(directory, self.testdir, symlinks=True)
 
+        # Check for a presense of a file with rules for ref
+        # either following special syntax ref: @filepath
+        if ref and str(ref)[0] == '@':
```

Comment:
https://github.com/teemtee/tmt/pull/1608 should remove the need for all those `str(ref)`, `ref` should be always a string internally, even with integer-like refs.

Reply:
So the fix is #1609, not #1608.

Reply:
I will create new PR removing those `str()` calls once your PR gets merged.